src/main/proto/ga4gh/common.proto

```diff
@@ -5,6 +5,16 @@ syntax = "proto3";
 
 package ga4gh;
 
+// When returning an HTTP error response, a server may also return a JSON formatted GAException
+// to better describe the error.
+message GAException {
+  // Numerical error code (should match the HTTP response code).
```

Comment:
Interesting.. I figured this was a safe guess. I guess that's the problem with this documentation being extremely vague in the first place.

Reverting sounds good, though I'd like to keep the comment above GAException since it's optional, and I think accurately reflects how it is being used currently.